spmd/compiler/fusion.py

```diff
@@ -72,10 +72,13 @@ class GraphInfo:
     fe_list: List[FusionElement] = field(default_factory=lambda: [])
     # max memory needed for fusion buffer
     peak_memory_required: int = 0
-    # node housing global buffer for fusion comms
-    global_buffer_node: Optional[fx.Node] = None
+    # list housing global buffers for fusion comms
+    _ring_buffer: Optional[List[fx.Node]] = None
     # size of the global buffer
     global_buffer_size: int = 0
+    _ring_size: int = 0
```

Comment:
nit: I would call this num_ring_buffs perhaps?

Reply:
good point - I've updated to _ring_num_buffers so that it's clear the variable here is the number of buffers in the ring.  (num_ring_buffs could allude to having multiple ring buffers).